lib/matplotlib/__init__.py

```diff
@@ -1690,7 +1690,7 @@ def param(func):
                 arg_names = []
             elif len(_arg_names) > 1 and (positional_parameter_names is None):
                 # we got no manual parameter names but more than an 'ax' ...
-                if len(set(replace_names) - set(_arg_names[1:])) == 0:
+                if len(replace_names - set(_arg_names[1:])) == 0:
```

Comment:
`replace_names` is already converted to a set [at the beginning of the function](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/__init__.py#L1623); it's not really needed in this PR though.